modules/serviceregistry/src/main/java/org/opencastproject/serviceregistry/impl/ServiceRegistryJpaImpl.java

```diff
@@ -1010,7 +1010,8 @@ protected JpaJob updateInternal(EntityManager em, JpaJob job) throws Persistence
     try {
       tx.begin();
       fromDb = em.find(JpaJob.class, job.getId());
-      originalJob = JpaJob.from(fromDb.toJob());
+      // MH-13675 This resets the job state in the 'job' variable back to the previous value so commenting it out
```

Comment:
I would just remove this entirely since all references to `originalJob` are now gone...  While you're at it, removing :1008 probably makes sense as well.

Reply:
Done! I had missed the originalJob declaration... Thanks!